docs/root/source/transaction-concepts.md

```diff
@@ -15,30 +15,57 @@ one might register an identity or a creative work. The things are often called
 "assets" but they might not be literal assets.
 
 BigchainDB supports divisible assets as of BigchainDB Server v0.8.0.
-That means you can create/register an asset with an initial quantity,
-e.g. 700 oak trees. Divisible assets can be split apart or recombined
-by transfer transactions (described more below).
+That means you can create/register an asset with an initial number of "shares."
+For example, A CREATE transaction could register a truckload of 50 oak trees.
+Each share of a divisible asset must be interchangeable with each other share;
+the shares must be fungible.
 
-A CREATE transaction also establishes, in its outputs, the conditions that must
-be met to transfer the asset(s). The conditions may also be associated with a
-list of public keys that, depending on the condition, may have full or partial
-control over the asset(s). For example, there may be a condition that any
-transfer must be signed (cryptographically) by the private key associated with a
-given public key. More sophisticated conditions are possible. BigchainDB's
-conditions are based on the crypto-conditions of the [Interledger Protocol
-(ILP)](https://interledger.org/).
+A CREATE transaction can have one or more outputs.
+Each output has an associated amount: the number of shares tied to that output.
+For example, if the asset consists of 50 oak trees,
+one output might have 35 oak trees for one set of owners,
+and the other output might have 15 oak trees for another set of owners.
+
+Each output also has an associated condition: the condition that must be met
+(by a TRANSFER transaction) to transfer/spend the output.
+BigchainDB supports a variety of conditions,
+a subset of the [Interledger Protocol (ILP)](https://interledger.org/)
+crypto-conditions. For details, see
+[the documentation about Inputs and Outputs](https://docs.bigchaindb.com/projects/server/en/latest/data-models/inputs-outputs.html).
+
+Each output also has a list of all the public keys associated
+with the conditions on that output.
+Loosely speaking, that list might be interpreted as the list of "owners."
+A more accurate word might be fulfillers, signers, controllers,
+or tranfer-enablers.
+See BigchainDB Server [issue #626](https://github.com/bigchaindb/bigchaindb/issues/626).
+
+A CREATE transaction must be signed by all the owners.
+(If you're looking for that signature,
+it's in the one "fulfillment" of the one input.)
```

Comment:
Perhaps we could somehow mention that the fulfillment is encoded, and hence the signature will not be visible? For instance, something along the lines:

Note that the fulfillment URI is encoded. One could to retrieve the signature, using a crypto-conditions library.

As an aside the fulfillment URI is encoded using the ASN.1 Distinguished Encoding Rules (DER), and the resulting bytes are encoded using `BASE64URL` as per [RFC 4648, Section 5](https://tools.ietf.org/html/rfc4648#section-5). More details at https://tools.ietf.org/html/draft-thomas-crypto-conditions-02#section-9

I am not sure how much details this section wishes to expose though.